Strategic-Initiatives.md

```diff
@@ -20,10 +20,10 @@ and have the support needed.
 | New Streams APIs    | [Jeremiah Senkpiel][fishrock123]                          | https://github.com/Fishrock123/bob, https://github.com/Fishrock123/socket               |
 | V8 Currency         | [Michaël Zasso][targos]                                   |                                                                                         |
 | Open Web Standards  | [Myles Borins][MylesBorins] + [Joyee Cheung][joyeecheung] | https://github.com/nodejs/open-standards                                                |
-| Python 3 & GYP      | [Sakthipriyan Vairamani][thefourtheye]                    | https://github.com/nodejs/TSC/issues/642                                                |
+| Python 3 & GYP      | [Sam Roberts][sam-github] + [Christian Clauss][cclauss]   | https://github.com/nodejs/TSC/issues/642                                                |
```

Comment:
https://github.com/nodejs/TSC/issues/642 is closed. Should it be reopened? Or replaced here with https://github.com/nodejs/node/issues/25789?